src/tiles/mod.rs

```diff
@@ -0,0 +1,89 @@
+mod storage;
+
+use bevy::{
+    math::UVec2,
+    prelude::{Bundle, Color, Component},
+};
+pub use storage::*;
+
+use crate::map::TilemapId;
+
+#[derive(Component, Default, Clone, Copy, Debug)]
```

Comment:
```suggestion
#[derive(Component, Default, Clone, Copy, Debug, Hash)]
```
My plugin makes liberal use of `HashMap<TilePos>`

Reply:
This is done.